ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightApiResult.kt

```diff
@@ -0,0 +1,63 @@
+import org.json.JSONObject
+
+/**
+ * Represents the result from the Shopper Insight API.
+ *
+ * @property eligibleMethods Contains the payment methods available to the shopper.
+ */
+data class ShopperInsightApiResult(
+    val eligibleMethods: PaymentMethods
+) {
+    companion object {
+        fun fromJson(jsonString: String): ShopperInsightApiResult {
```

Comment:
Let's restrict this to library scope - merchants shouldn't need to call this

Reply:
I updated to make ShopperInsightApiResult as internal. Merchants will only have `ShopperInsightsInfo` returned to them when they make a call to ShoppingInsightsClient.